apm-agent-plugins/apm-vertx/apm-vertx4-plugin/src/main/java/co/elastic/apm/agent/vertx/v4/web/NoopVertxTracer.java

```diff
@@ -0,0 +1,47 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2021 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.vertx.v4.web;
+
+import io.vertx.core.Context;
+import io.vertx.core.spi.tracing.SpanKind;
+import io.vertx.core.spi.tracing.TagExtractor;
+import io.vertx.core.spi.tracing.VertxTracer;
+import io.vertx.core.tracing.TracingPolicy;
+
+class NoopVertxTracer implements VertxTracer<Object, Object> {
```

Comment:
[minor] We could simply reuse the `io.vertx.core.spi.tracing.VertxTracer#NOOP` constant instead

Reply:
Good point, missed there is a NOOP tracer.

Reply:
Tried to do the same with the `io.vertx.core.spi.tracing.VertxTracer#NOOP` tracer. Unfortunately, `NOOP` is returning null as the trace object so that the corresponding `sendResponse` method of the traces isn't called at all. 
--> We need to use our own NoopVertxTracer!